create-publication-from-scopus/src/test/java/no/sikt/nva/scopus/ScopusHandlerTest.java

```diff
@@ -254,11 +287,35 @@ void shouldReturnDefaultPublicationContextWhenEventWithS3UriThatPointsToScopusXm
         assertThat(actualJournalName, is(HARD_CODED_JOURNAL_NAME_IN_RESOURCE_FILE));
     }
 
+    @Test
+    void shouldReturnCreatePublicationRequestWithJournalWhenEventWithS3UriThatPointsToScopusXmlWhereSourceTitleIsInNsd()
```

Comment:
I am a bit confused, because now we are setting up Wiremock but we never use it and we are mocking the HttpClient instead. Since Wiremock is not in use we should remove it.

Reply:
What? Let me check again. The meaning was to keep the wiremock ...

Reply:
Sorry, my mistake. I wasn't carefull enough. I deleted the wrong test method.
The wiremock using test method is back now.